aws-cloudformation/cfn-lint
# Test format of json files

*Issue #, if available:*

*Description of changes:*
- Test format of json files
- Add script to format json files as needed
- Run black on scripts/
- Run isort on scripts/

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
